tests/Rules/NoModelForwardingToBuilderTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Rules;
+
+use Larastan\Larastan\Rules\NoModelForwardingToBuilder;
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+/** @extends RuleTestCase<NoModelForwardingToBuilder> */
+class NoModelForwardingToBuilderTest extends RuleTestCase
+{
+    protected function getRule(): Rule
+    {
+        return new NoModelForwardingToBuilder();
+    }
+
+    public function testRuleDetectsModelForwardingToBuilderInstance(): void
```

Comment:
just a nitpick, but these methods can just be called `testRule` (pairs well with `getRule`)